### PR TITLE
Feature dubbo hijack

### PR DESCRIPTION
**Pull request by champly**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

**Comment by cch123** [comment]
cc @zonghaishang

File: pkg/protocol/xprotocol/dubbo/types.go [hunk 1]
```diff
@@ -63,3 +68,33 @@ const (
 const (
 	ResponseStatusSuccess uint16 = 0x14 // 0x14 response status
 )
+
+var (
+	dubboStatusMsg = map[int]string{
```
Comment (cch123) [inline pkg/protocol/xprotocol/dubbo/types.go:73]: merge this dubboStatusMap with dubboStatusMsg

**Comment by cch123** [comment]
LGTM

**Comment by champly** [comment]
#1236

**Comment by cch123** [comment]
add a test case for hijack && a case for non-hijack

**Comment by champly** [comment]
> add a test case for hijack && a case for non-hijack

already add unit test

**Pull request merged**